projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -64,13 +76,18 @@ import { TableColumnConfigExtended } from '../table.service';
             </div>
           </ht-popover-content>
         </ht-popover>
+
+        <ng-template #htmlTooltip>
+          <div [innerHTML]="this.sanitizedHtmlForTooltip"></div>
+        </ng-template>
       </ng-template>
     </div>
   `
 })
 export class TableHeaderCellRendererComponent implements OnInit, OnChanges {
   public readonly SORT_ASC: TableSortDirection = TableSortDirection.Ascending;
   public readonly SORT_DESC: TableSortDirection = TableSortDirection.Descending;
+  private readonly HTML_TOOLTIP_PREFIX: string = 'html:';
```

Comment:
why the prefix? even a plain string is valid html so can we treat any tooltips that way?

Reply:
I was being _clever_ with adding a differentiator to html strings. But you are right, its unnecessary. Removed.